internal/ethapi/api.go

```diff
@@ -592,14 +592,23 @@ func (s *PublicBlockChainAPI) GetProof(ctx context.Context, address common.Addre
 		codeHash = crypto.Keccak256Hash(nil)
 	}
 
+	// toHexArray creates a array of hex-string based on []byte
```

Comment:
I would prefer this to be a globally-defined unexported function in this package.